R/geom_phylopic.R

```diff
@@ -226,8 +226,11 @@ GeomPhylopic <- ggproto("GeomPhylopic", Geom,
     data$img <- imgs
     data
   },
-  use_defaults = function(self, data, params = list(), modifiers = aes()) {
-    default_aes <- self$default_aes
+  use_defaults = function(self, data, params = list(), modifiers = aes(),
+                          default_aes = NULL, theme = NULL, ...) {
+    if (packageVersion("ggplot2") < "4.0.0") {
+      default_aes <- self$default_aes
+    }
     # Inherit size as height if no height aesthetic and param exist
     if (!is.null(data$size)) {
       lifecycle::deprecate_warn("1.5.0",
```

Comment:
Which version do we want to stage removing lifecycle for?

Reply:
Hmm...I guess we've had this warning for a year now. We could progress it to an error (with `deprecate_stop()` for this release (https://lifecycle.r-lib.org/articles/communicate.html#gradual-deprecation) and then fully remove the code (and dependency if we don't need it) in the next release (no idea when that would be)?